packages/datetime/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "@blueprintjs/datetime",
-  "version": "1.3.0",
+  "version": "1.4.0",
   "description": "Components for interacting with dates and times",
   "main": "dist/index.js",
   "typings": "dist/index.d.ts",
   "style": "dist/blueprint-datetime.css",
   "dependencies": {
-    "@blueprintjs/core": "^1.3.0",
+    "@blueprintjs/core": "^1.4.0",
```

Comment:
I think you can revert this line too; it is not necessary to change the minimum dependency on core